crates/ruff_python_parser/src/token.rs

```diff
@@ -922,117 +922,116 @@ impl TryFrom<TokenKind> for UnaryOp {
     }
 }
 
-impl fmt::Display for TokenKind {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let value = match self {
-            TokenKind::Unknown => "<Unknown>",
-            TokenKind::StartModule => "StartProgram",
+impl TokenKind {
+    pub(crate) fn display(self) -> &'static str {
+        match self {
+            TokenKind::Unknown => "Unknown",
+            TokenKind::StartModule => "StartModule",
             TokenKind::StartExpression => "StartExpression",
             TokenKind::StartInteractive => "StartInteractive",
```

Comment:
These kinds are a bit awkward but I don't know of a better solution (I think ideally we would not have a display representation for those but it's too late when writing the error message to make assumption about what the token kind is.